lib/tagging/tagging_update_form.rb

```diff
@@ -0,0 +1,46 @@
+# Tagging update form
+#
+# This class is partially duplicated from content-tagger. Once we've added the
```

Comment:
Can we extract the form now? Is the only thing that differs the available link_types?